internal/serverinstall/k8s.go

```diff
@@ -309,6 +311,8 @@ func (i *K8sInstaller) Install(
 		// Ensure the service is ready to use before returning
 		_, err = net.DialTimeout("tcp", httpAddr, 1*time.Second)
 		if err != nil {
+			// Depending on the platform, this can take a long time. On EKS, it's by far the longest step. Adding an explicit message helps
+			s.Update("Service %s exists and is configured, but isn't yet accepting incoming connections. Waiting...\n", serviceName)
```

Comment:
```suggestion
			s.Update("Service %q exists and is configured, but isn't yet accepting incoming connections. Waiting...", serviceName)
```

Not sure if you need the `\n`, `Update` should get a new line by default unless you were hoping for an extra one. We should probably also update the stepgroup for when the service is ready and we're done waiting?